src/test/scala/scraml/libs/RefinedSupportSpec.scala

```diff
@@ -0,0 +1,41 @@
+package scraml.libs
+
+import java.io.File
+
+import cats.effect.unsafe.implicits.global
+import org.scalatest.diagrams.Diagrams
+import org.scalatest.wordspec.AnyWordSpec
+import scraml.{DefaultModelGen, DefaultTypes, ModelGenParams, ModelGenRunner}
+
+class RefinedSupportSpec extends AnyWordSpec with Diagrams {
```

Comment:
The spec is a bit undersized given the implementation, I assume you want to iterate a bit more on the support before fixating its behavior?